templates/todomvc/src/main.js

```diff
@@ -1,11 +1,77 @@
+let near = null;
+
+async function init() {
+  let getConfig = null;
+  if (window.__near_getConfig) {
+    getConfig = __near_getConfig;
+  } else {
+    getConfig = async function () {
+      return JSON.parse(decodeURIComponent(getCookie("fiddleConfig")));
+    }
+  }
+
+  studioConfig = await getConfig();
+  near = window.nearLib.Near.createDefaultConfig(studioConfig.nodeUrl);
+  myAccountId = window.localStorage.getItem("nearstudio_account_id");
+  near.getConfig = getConfig;
+  near.requireDefault = async function(viewMethods, changeMethods) {
+    let nearConfig = await this.getConfig();
+    return this.require(nearConfig.contractName, viewMethods, changeMethods);
+  }
+
+  near.require = async function(contractName, viewMethods, changeMethods) {
+    let nearConfig = await this.getConfig();
+
+    // TODO: Introspection of contract methods
+    let contract = {};
+    viewMethods.forEach((methodName) => {
+      contract[methodName] = async function(args) {
+        //console.log(methodName, args);
+        args = args || {};
+        let response = await sendJson("POST", `${nearConfig.baseUrl}/contract/view/${contractName}/${methodName}`, { args });
```

Comment:
this still goes around nearlib

Reply:
Not sure what you mean by this

Reply:
I mean that this code calls contract helper. It should be calling nearlib instead.